recipes/r-vioplot/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# R refuses to build packages that mark themselves as Priority: Recommended
```

Comment:
Please remove these extra lines. `build.sh` should only be the following two lines:

```
#!/bin/bash
$R CMD INSTALL --build .
```

You can add the string "[ci skip] [skip ci]" to your commit message to avoid re-triggering the CI builds for this cosmetic change.